modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java

```diff
@@ -20,6 +20,7 @@
 import com.liferay.object.field.setting.util.ObjectFieldSettingUtil;
 import com.liferay.object.model.ObjectDefinition;
 import com.liferay.object.model.ObjectField;
+import com.liferay.object.model.ObjectRelationship;
 import com.liferay.object.rest.dto.v1_0.FileEntry;
 import com.liferay.object.rest.dto.v1_0.ListEntry;
```

Comment:
I tested this functionality with 1toM and MtoM in an Object with himself and it does not include the related schema in the fields, could you review why? maybe we should create a bug for it or solve it in this PR

Reply:
I will take a view at this, but I think that I would fix it in this PR